easybuild/easyblocks/t/tensorflow.py

```diff
@@ -760,6 +771,14 @@ def build_step(self):
         self.bazel_opts = [
             '--output_user_root=%s' % self.output_user_root_dir,
         ]
+        if bazel_version >= '4.0.0':
+            self.bazel_opts.append('--local_startup_timeout_secs=600')  # 5min for bazel to start
+
+        # Environment variables and values needed for Bazel actions.
+        action_env = {}
+        # A value of None is interpreted as using the invoking environments value
+        INHERIT = None  # For better readability
```

Comment:
Defining a local constant like this is a bit weird tbh. Why not just use a local variable like `inherit_env`?

If you strongly prefer a constant, define it outside of the `class`, on top of `tensorflow.py` (and maybe rename to `INHERIT_ENV`)

Reply:
Basically I had `action_env['PYTHONPATH'] = None` which wasn't really clear. `action_env['PYTHONPATH'] = INHERIT` reads much better and defining the constant next to where it has meaning (i.e. `action_env`) should avoid misuse.

It should be a constant (hence uppercase by convention and IIRC linters) not a variable and local because outside this method it doesn't has a purpose. So I'd prefer to keep it as is. I also don't see a benefit of `action_env['PYTHONPATH'] = INHERIT_ENV` over `action_env['PYTHONPATH'] = INHERIT` but find the former more "noisy".

If you feel strongly about changing this I will as in the end it's personal preference I guess. The current version is just what I find best fitting but won't fight for it besides explaining my reasoning (above)

Reply:
OK, clear, thanks for the detailed motivation, makes sense